modules/grpc/protos/googleapis-proto/google/pubsub/v1/schema.proto

```diff
@@ -0,0 +1,410 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
```

Comment:
Is this generated code? It might worth adding to a readme how it was generated in case someone needs to redo it in the future.

Reply:
No, this comes from https://github.com/googleapis/googleapis/blob/master/google/pubsub/v1/pubsub.proto.

I am adding a note about it to the description.

Reply:
Done.